compiler/damlc/lib/DA/Cli/Damlc/Command/Damldoc.hs

```diff
@@ -136,6 +138,30 @@ documentation = Damldoc
         long "combine"
         <> help "Combine all generated docs into a single output file (always on for json and hoogle output)."
 
+    optExtractOptions :: Parser ExtractOptions
+    optExtractOptions = ExtractOptions
+        <$> optQualifyTypes
+        <*> optSimplifyQualifiedTypes
+
+    optQualifyTypes :: Parser QualifyTypes
+    optQualifyTypes = option readQualifyTypes $
+        long "qualify-types"
+        <> metavar "MODE"
+        <> help "Qualify any non-local types in generated docs. Can be set to \"always\" (always qualify non-local types), \"never\" (never qualify non-local types), and \"inpackage\" (qualify non-local types defined in the same package). Defaults to \"never\"."
+
+    readQualifyTypes =
+        eitherReader $ \arg ->
+            case lower arg of
+                "always" -> Right QualifyTypesAlways
+                "inpackage" -> Right QualifyTypesInPackage
+                "never" -> Right QualifyTypesNever
+                _ -> Left "Unknown mode for --qualify-types. Expected \"always\", \"never\", or \"inpackage\"."
+
+    optSimplifyQualifiedTypes :: Parser Bool
+    optSimplifyQualifiedTypes = switch $
```

Comment:
It looks like this option doesn’t do anything if you don’t also qualify types. Should we emit a warning if users only specify `--simplify-qualified-types`?